app/assets/stylesheets/finder_frontend.scss

```diff
@@ -233,16 +233,91 @@
     .result-info {
       @include core-16;
       vertical-align: baseline;
-      border-bottom: solid 1px $border-colour;
       margin: 0 0 $gutter-half 0;
-      padding-bottom: $gutter-half;
 
       .result-count {
         @include bold-48;
         margin-right: $gutter-one-third / 2;
       }
     }
 
+    .filters {
+      margin-top: $gutter-one-third;
+      margin-bottom: $gutter;
+    }
+
+    .facet-tag-group {
+      padding: 5px;
+
+      &:nth-child(odd) {
+        background-color: $grey-4;
+      }
+
+      .facet-tag-wrapper {
+        vertical-align: middle;
+        margin: 5px 0;
+
+        @include media(tablet) {
+          display: inline-block;
+        }
+      }
+
+      .facet-tag-wrapper:first-of-type .facet-tag-preposition:first-of-type {
+        font-weight: bold;
+
+        &::first-letter {
+          text-transform: uppercase;
+        }
+      }
+
+      .facet-tag-preposition {
+        padding: 0 5px 0 2px;
+        vertical-align: middle;
```

Comment:
Spacing between this and the facet tag on mobile is quite close, might be good to space it out a little more.

![screen shot 2018-12-18 at 16 27 34](https://user-images.githubusercontent.com/861310/50167858-fd2d1c80-02e1-11e9-922b-e919004b5659.png)

Reply:
Good spot 👍

Reply:
Done